news/2017-09-03-qat-gazette-september-2017.md

```diff
@@ -0,0 +1,57 @@
+
+There were highlights of upcoming projects and events, reviews of past occurrences such as Beatmap Nominator tests and the Best of 2016 awards, and sometimes opinion pieces simply giving our thoughts on various aspects of the community, the mapping meta, and so on. With the growth of the mapping and modding community over the past year, as well as the significant changes to the role of the QAT, more and more people rightfully ask, **“What the heck do you guys actually do now!?”** Hence the birth of regular updates, and now, a move into the mainstream for all to see!
+
+## Updates to the Beatmap Nomination Group
+
+Firstly, **please welcome two new Beatmap Nominators to the Advanced Tier for osu!standard - Battle and Naxess!** As you may be aware, Advanced Tier Nominators are responsible for beatmap qualification, and with the recent departure of a number of existing members, it’s important that receiving a heart icon doesn’t become a bottleneck for mappers. Please treat them well as they settle into their new role!
+
+On the other side of the coin, **a number of existing members of the Beatmap Nomination Group will be departing** to pursue other opportunities. Many thanks to Dellvangel, -Kamikaze-, Kuo Kyoka, newyams99, Nivrad00 and Sonnyc for your contributions to our community over the months and years!
+
+Fear not however, as **we are currently reviewing applications for a new batch of fresh-faced modders to replenish their ranks!** The latest application process for the osu!taiko, osu!catch and osu!mania! game modes has recently closed, with successful applicants expected to promoted in the coming weeks.
+
+With the discontinuation of Beatmap Nominator tests, new BN applications will consist of an activity check, as well as submission of the applicant’s “best” modding work - either as a standard mod post, or series of suggestions in the moddingv2 panel. 
+
+For those that meet the activity requirements, the QAT will then supplement this with additional random, anonymised examples of the applicant’s modding work from the past three months and evaluate these examples on a number of criteria, such as structure, helpfulness, knowledge of the ranking criteria, and so on. These modding examples will be voted on by the remaining members of the QAT who were not involved in selecting examples of applicant modding, to avoid bias.
+
+**Those with a majority of positive votes will be promoted into one of the available** [Nominator Subdivisions](https://osu.ppy.sh/forum/p/6201585) and closely surveyed on their general attitude (forum posts, in-game chat, modding and icon activity, etc.) to ensure they receive the assistance needed to fulfill their roles as gatekeepers of content in the mapping and modding community.
```

Comment:
could mention that feedback on their applications will stay an on demand thing as it is today additionally

Reply:
Added sentence regarding feedback